app/routers/expenses.py

```
@@ -11,15 +11,55 @@
+    size = await file.read()
+    if len(size) > MAX_FILE_SIZE:
+        raise HTTPException(
+            status_code=status.HTTP_413_PAYLOAD_TOO_LARGE,
+            detail=f"File size exceeds the limit of {MAX_FILE_SIZE // 1024 // 1024}MB."
+        )
+    await file.seek(0)
+
+    try:
+        image_data = await file.read()
+        db_expenses = create_expense_from_receipt(db=db, image_data=image_data, user_id=current_user.id)
+        return db_expenses
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)